### PR TITLE
Add restic check handling

### PR DESCRIPTION
**Pull request by skimpax**
Add support of 'restic check' command in resticker, update examples and Readme accordingly.

### CODE REVIEW

File: backup [hunk 1]
```diff
@@ -29,7 +29,7 @@ start=`date +%s`
 echo Starting Backup at $(date +"%Y-%m-%d %H:%M:%S")
 
 set +e
-restic backup ${RESTIC_BACKUP_ARGS:-} ${tag_options} ${RESTIC_BACKUP_SOURCES}
+restic --repo="${RESTIC_REPOSITORY}" backup ${RESTIC_BACKUP_ARGS:-} ${tag_options} ${RESTIC_BACKUP_SOURCES}
```
Comment (djmaze) [inline backup:32]: Is this just to make it more obvious?

**Review by djmaze — APPROVED** [review]
LGTM and works for me. Thanks, good work and a very nice addition!

File: backup (hunk 1 above)
Comment (skimpax) [reply backup:32]: Yes.
Motivation was that `check` command  based on exiting format was raising a restic error.
It seems 'restic check' does not parse parameters the same way as 'restic backup`.
To cope with this, I changed the syntax to be closer to restic documentation examples... and its more obvious, indeed.

**Comment by djmaze** [comment]
Okay, sorry for the late response. Merging.

**Pull request merged**